megamek/src/megamek/common/alphaStrike/ASArcs.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2022 - The MegaMek Team. All Rights Reserved.
+ *
+ * This file is part of MegaMek.
+ *
+ * MegaMek is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * MegaMek is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with MegaMek. If not, see <http://www.gnu.org/licenses/>.
+ */
+package megamek.common.alphaStrike;
+
+public enum ASArcs {
+    FRONT, LEFT, RIGHT, REAR;
+
+    @Override
+    public String toString() {
+        switch (this) {
+            case LEFT:
+                return "Left Arc";
+            case RIGHT:
+                return "Right Arc";
+            case REAR:
+                return "Rear Arc";
+            default:
+                return "Front Arc";
+        }
+    }
+
+    public int toInt() {
+        switch (this) {
+            case LEFT:
+                return 1;
+            case RIGHT:
+                return 2;
+            case REAR:
+                return 3;
+            default:
+                return 0;
+        }
+    }
```

Comment:
This is just `.ordinal()` ?

Reply:
It is, but if someone changed the order, conversion would no longer work correctly. I think I'll move this over to the conversion code, it's less out of place there.